hpy/tools/autogen/public_api.h

```diff
@@ -115,30 +115,30 @@ HPy_ID(79)
 void HPy_Close(HPyContext *ctx, HPy h);
 
 HPy_ID(80)
-HPy HPyLong_FromLong(HPyContext *ctx, long value);
+HPy HPyLong_FromInt32_t(HPyContext *ctx, int32_t value);
 HPy_ID(81)
-HPy HPyLong_FromUnsignedLong(HPyContext *ctx, unsigned long value);
+HPy HPyLong_FromUInt32_t(HPyContext *ctx, uint32_t value);
 HPy_ID(82)
-HPy HPyLong_FromLongLong(HPyContext *ctx, long long v);
+HPy HPyLong_FromInt64_t(HPyContext *ctx, int64_t v);
 HPy_ID(83)
-HPy HPyLong_FromUnsignedLongLong(HPyContext *ctx, unsigned long long v);
+HPy HPyLong_FromUInt64_t(HPyContext *ctx, uint64_t v);
 HPy_ID(84)
 HPy HPyLong_FromSize_t(HPyContext *ctx, size_t value);
```

Comment:
Since `size_t` is implementation-defined, do we want to do the same for it as for `int`/`long`?

Reply:
That's a point and we probably should. This basically means that we would remove `ctx_Long_FromSize_t` from the context and just have an inline helper for it (which, I think, is fine).